bundle/config/mutator/apply_presets_test.go

```diff
@@ -125,6 +125,36 @@ func TestApplyPresetsPrefixForUcSchema(t *testing.T) {
 	}
 }
 
+func TestApplyPresetsUCVolumesShouldNotBePrefixed(t *testing.T) {
```

Comment:
Style: mix of `UC` and `Uc` in test names. Can be omitted as UC is implied.

Reply:
fixed